tests/docker-images/statestore-image/Dockerfile

```diff
@@ -46,8 +46,8 @@ RUN mkdir /opt/bookkeeper/conf
 RUN mkdir /opt/bookkeeper/scripts
 
 ### -----Copy Jars------###
-ADD ./dist/server.tar.gz /opt/
-RUN mv /opt/server/lib/*.jar /opt/bookkeeper/lib/
+ADD ./dist/bookkeeper-server-*.*SNAPSHOT.tar.gz /opt/
```

Comment:
We should we able to run the tests on the source release, because people would want to build and run tests from the source tarball.
This is very important for a ASF project

Reply:
Oh I see. I think we are not building src tar ball I think.